src/pages/gen2/start/quickstart/nuxtjs-app/index.mdx

```diff
@@ -0,0 +1,309 @@
+export const meta = {
+  title: 'Nuxt 3 App',
+  description: 'Get started with AWS Amplify (Gen 2) using Nuxt.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Nuxt, and Vue. If you are new to these technologies, we recommend you go through the official [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html), [Nuxt](https://nuxt.com/docs/getting-started/introduction), and [Vue](https://vuejs.org/guide/introduction.html) tutorials first.
+
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+
+<Fragments fragments={{ javascript: prerequisites, nuxtjs: prerequisites }} />
+
+import createProject from 'src/fragments/gen2/quickstart/create-nuxtjs-project.mdx';
+
+<Fragments fragments={{ javascript: createProject, nuxtjs: createProject }} />
+
+import createAmplify from 'src/fragments/gen2/quickstart/create-amplify.mdx';
+
+<Fragments fragments={{ javascript: createAmplify, nuxtjs: createAmplify}} />
+
+import buildABackend from 'src/fragments/gen2/quickstart/build-a-backend.mdx';
+
+<Fragments fragments={{ javascript: buildABackend, nuxtjs: buildABackend }} />
+
+
+## Build UI
+
+Let's add UI that connects to the backend data and auth resources.
+
+### Add a login form
+
+First, install the Amplify UI component library:
+
+```bash title="Terminal" showLineNumbers={false}
+npm install @aws-amplify/ui-vue
+```
+
+Next, in your app's `app.vue` file, add the following imports and wrap your App in the `Authenticator` function:
+
+```tsx title="nuxt-amplify-gen2/app.vue"
+// app.vue
+<script setup lang="ts">
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
+import { Amplify } from 'aws-amplify';
+import config from '@/amplifyconfiguration.json';
+
+// configure the Amplify client library with the configuration generated by `amplify sandbox`
+Amplify.configure(config);
+const auth = useAuthenticator();
+
+</script>
+
+
+<template>
+  <Authenticator>
+    <template v-if="auth.route === 'authenticated'">
+      <h1>Hello, Amplify 👋</h1>
+      <button @click="auth.signOut">Sign Out</button>
+    </template>
+  </Authenticator>
+</template>
+```
+
+Run your application with `npm run dev` and navigate to `http://localhost:3000`. You should now see the authenticator, which is already configured and ready for your first sign-up! Create a new user account, confirm the account through email, and then sign in.
+
+### View list of to-do items
+
+Now, let's display data on our app's frontend. Modify your app's home page file, `app.vue`, with the following code:
+
+```diff title="nuxt-amplify-gen2/app.vue"
+// app.vue
+<script setup lang="ts">
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
```

Comment:
We have a [Nuxt with Server Side Rendering Guide](https://docs.amplify.aws/nextjs/build-a-backend/server-side-rendering/nuxt/#pageMain) that recommends for the user to create an Amplify Plugin. I would prefer we recommend this approach, or at the least, have a warning that describes that this getting started will only work for client side apps.